src/main/java/com/gregtechceu/gtceu/common/machine/multiblock/steam/SteamParallelMultiblockMachine.java

```diff
@@ -44,13 +44,14 @@
 @MethodsReturnNonnullByDefault
 public class SteamParallelMultiblockMachine extends WorkableMultiblockMachine implements IDisplayUIMachine {
 
-    public static final int MAX_PARALLELS = 8;
+    public int MAX_PARALLELS = ConfigHolder.INSTANCE.machines.steamMultiParallelAmount;
 
     // if in millibuckets, this is 0.5, Meaning 2mb of steam -> 1 EU
     private static final double CONVERSION_RATE = 0.5D;
 
-    public SteamParallelMultiblockMachine(IMachineBlockEntity holder, Object... args) {
-        super(holder, args);
+    public SteamParallelMultiblockMachine(IMachineBlockEntity holder, int parallelAmount) {
+        super(holder, parallelAmount);
+        MAX_PARALLELS = parallelAmount;
     }
```

Comment:
```suggestion
    public SteamParallelMultiblockMachine(IMachineBlockEntity holder, int parallelAmount) {
        super(holder);
        maxParallels = parallelAmount;
    }
```
Keep the super constructor clean in case of future schema changes. The super doesn't need to know about the parallel amount.